FoxHound.Client/src/Blogs/Blogs.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useState, useEffect, ChangeEvent } from "react";
-import Blog from "./Blog";
+import Blog from "./Blog/Blog";
 import axios from "axios";
-import AddBlog from "./AddBlog";
+import AddBlog from "./Blog/AddBlog";
```

Comment:
I think I would just have a higher level Blogs folder and not the nested Blog folder within it. This is just a suggestion. My reasoning being that I don't think there's enough blog related components that we'll be building that warrant multiple folders for them. I think for now, it may also lead to confusion as to where do I put a new component related to a blog.